tfsec.yaml

```diff
@@ -0,0 +1,58 @@
+package:
+  name: tfsec
+  version: 1.28.5
+  epoch: 0
+  description: Security scanner for your Terraform code
+  copyright:
+    - license: MIT
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - git
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 2a2a67987b477eb4a76ff42858973096138e9943
+      repository: https://github.com/aquasecurity/tfsec
+      tag: v${{package.version}}
+
+  - uses: go/bump
+    with:
+      deps: google.golang.org/protobuf@v1.33.0
+
+  - runs: |
+      export BINARY=tfsec
+      export CHECK_GEN_BINARY=tfsec-checkgen
+      export TAG=v${{package.version}}
+      export GO111MODULE=on
+      export CGO_ENABLED=0
+      export GOOS=$(go env GOOS)
+      export GOARCH=$(go env GOARCH)
+      mkdir -p bin/${GOOS}
+      go build \
```

Comment:
try to use go/build where possible. this way, if we want to make change to build flag later, it'll be easier

https://github.com/chainguard-dev/melange/blob/main/pkg/build/pipelines/go/build.yaml

Reply:
go/build also has a flag for bumping dep